ansible_collections/arista/avd/roles/eos_designs/python_modules/connected_endpoints/port_channel_interfaces.py

```diff
@@ -175,6 +175,11 @@ def _get_port_channel_interface_cfg(self, adapter: dict, port_channel_interface_
         elif self.shared_utils.mlag and len(set(adapter["switches"])) > 1:
             port_channel_interface["mlag"] = channel_group_id
 
+        # Set ptp mpass on port-channel if ptp is enabled and this switch is running MLAG
```

Comment:
In the `elif` above we have all the checks for mlag, so you could just move the logic under that. Then it will only trigger on mlag port-channels. But the two ptp checks you have on the same line.

Reply:
Done!